templates/repo/issue/view_title.tmpl

```diff
@@ -37,11 +37,40 @@
 		{{else}}
 			{{if .Issue.OriginalAuthor }}
 				{{.Issue.OriginalAuthor}}
-				<span class="pull-desc">{{$.i18n.Tr "repo.pulls.title_desc" .NumCommits .HeadTarget .BaseTarget | Str2html}}</span>
+				<span id="pull-desc" class="pull-desc">{{$.i18n.Tr "repo.pulls.title_desc" .NumCommits .HeadTarget .BaseTarget | Str2html}}</span>
 			{{else}}
 				<a {{if gt .Issue.Poster.ID 0}}href="{{.Issue.Poster.HomeLink}}"{{end}}>{{.Issue.Poster.GetDisplayName}}</a>
-				<span class="pull-desc">{{$.i18n.Tr "repo.pulls.title_desc" .NumCommits .HeadTarget .BaseTarget | Str2html}}</span>
+				<span id="pull-desc" class="pull-desc">{{$.i18n.Tr "repo.pulls.title_desc" .NumCommits .HeadTarget .BaseTarget | Str2html}}</span>
 			{{end}}
+			<span id="pull-desc-edit" style="display: none">
+                 <div class="ui floating filter dropdown" data-no-results="{{.i18n.Tr "repo.pulls.no_results"}}">
+                	<div class="ui basic small button">
+                		<span class="text" id="pull-target-branch" data-basename="{{$.BaseName}}" data-branch="{{$.BaseBranch}}">{{.i18n.Tr "repo.pulls.compare_base"}}: {{$.BaseName}}:{{$.BaseBranch}}</span>
+                		<i class="dropdown icon"></i>
+                	</div>
+                	<div class="menu">
```

Comment:
As we also have this select of source and target branch on the regular pull request page, we should put it inside a partial and use that on the pull request form and here. By passing settings to the partial we'd be able to chose which field should be editable.

Reply:
Proposal scrapped because I changed the order of the select fields in order to match the wording in the text.